### ISSUE TITLE
Issue [Running Bot]

### ISSUE THREAD

**Issue by JoinZCoin** [MSFTserver/TipBot#28]
I have the bot loaded and it showing in the discord.

The following message was shown in discord:

```
[05-15-2019 05:43 am PST][TipBot] CoinJoinLogged in! Serving in 1 servers
[05-15-2019 05:43 am PST][TipBot] type !tiphelp in Discord for a commands list.
```
in PM2 log
```
1|CoinJoin | Loaded 2 chat commands and 0 custom functions.
1|CoinJoin | [05-15-2019 05:43 am PST][TipBot] type !tiphelp in Discord for a commands list.
```
When the command is given !tiphelp nothing comes back in discord 

PM2 screen

```
status            │ online          
│ name              │ CoinJoin 
│ version           │ 1.0.0   
│ restarts          │ 2455   
│ uptime            │ 2m   
│ script path       │ /home/Desktop/TipBot-master/bot/bot.js 
│ script args       │ N/A                                               │
│ error log path    │ /home/Desktop/TipBot-master/tipbot-err.log │
│ out log path      │ /home/Desktop/TipBot-master/tipbot-out.log │
│ pid path          │ /home/.pm2/pids/CoinJoin-1.pid             │
│ interpreter       │ node                                              │
│ interpreter args  │ N/A                                               │
│ script id         │ 1                                                 │
│ exec cwd          │ /home/Desktop/TipBot-master                │
│ exec mode         │ fork_mode                                         │
│ node.js version   │ 8.11.4                                            │
│ node env          │ development                                       │
│ watch & reload    │ ✔                                                 │
│ unstable restarts │ 0                                                 │
│ created at        │ 2019-05-15T12:43:46.330Z                          │
└───────────────────┴───────────────────────────────────────────────────┘
 Code metrics value 
┌────────────────┬───────────┐
│ Heap Size      │ 23.39 MiB │
│ Heap Usage     │ 70.42 %   │
│ Used Heap Size │ 16.47 MiB │

```

If I do the command !tiprvn the bot will PM me but won't send me the help commands

Any advice would be great!

**Comment by JoinZCoin** [comment]
On further inspection it seems there is some issue with the plugins.

I am able to call the command !tiprvn and returns in PM fine but no other commands are working at this time.

`0|CoinJoin | Improper setup of the 'dogeTipper.js' plugin. : Error: Configuration property "doge" is not defined
0|CoinJoin | Improper setup of the 'ftcTipper.js' plugin. : Error: Configuration property "ftc" is not defined
0|CoinJoin | Improper setup of the 'helpTipper.js' plugin. : Error: Configuration property "doge" is not defined
0|CoinJoin | Improper setup of the 'lbcTipper.js' plugin. : Error: Configuration property "lbc" is not defined
0|CoinJoin | Improper setup of the 'neblTipper.js' plugin. : Error: Configuration property "nebl" is not defined
0|CoinJoin | Improper setup of the 'phaseTipper.js' plugin. : Error: Configuration property "phase" is not defined
0|CoinJoin | Improper setup of the 'ppcTipper.js' plugin. : Error: Configuration property "ppc" is not defined
0|CoinJoin | Improper setup of the 'pxcTipper.js' plugin. : Error: Configuration property "pxc" is not defined
0|CoinJoin | Improper setup of the 'spkTipper.js' plugin. : Error: Configuration property "spk" is not defined
0|CoinJoin | Improper setup of the 'ufoTipper.js' plugin. : Error: Configuration property "ufo" is not defined
0|CoinJoin | Improper setup of the 'vtlTipper.js' plugin. : Error: Configuration property "vtl" is not defined
0|CoinJoin | Improper setup of the 'xuezTipper.js' plugin. : Error: Configuration property "xuez" is not defined
0|CoinJoin | Loaded 2 chat commands and 0 custom functions.
0|CoinJoin | [05-15-2019 10:34 am PST][CoinJoin] type !tiphelp in Discord for a commands list.
0|CoinJoin | treating !tiprvn from --User-- as command
`

**Comment by MSFTserver** [comment]
remove the files for the coins your not using or add the config options for them

**Comment by MSFTserver** [comment]
could also be in the help command file where it tries to get the fees

**Comment by JoinZCoin** [comment]
Thank you for the speedy response. 
I solved this issue but I can't seem to get the bot to return address it just fails with :

Error getting your Bitcoin (BTC) deposit address.

I have included the specific RCP commands in bitcoin.conf

Bitcoin is listening on RPC 8332 on localhost but there seems to be no connection between bitcoin core and the bot.

```
Example of Bitcoin.conf
rpcpassword=
rpcuser=
server=1
rpcbind=127.0.0.1
rpcallowip=127.0.0.1
rpcport=8332
rpcclienttimeout=30
rpcthreads=2
rpcworkqueue=1000
paytxfee=0.00001
```


Example default.conf


```
  },
  "btc": {
    "paytxfee": .00001,
    "config": {
      "port": 8332,
      "user": "",
      "pass": ""
    }
  },
```
  
I also have a warning in the log file showing 

`WARNING: NODE_APP_INSTANCE value of '0' did not match any instance config file names.`
